coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/RelToTrinoConverter.java

```diff
@@ -98,6 +99,7 @@ public Result visit(Window window) {
     return null;
   }
 
+  // override is required to prevent select nodes such as SELECT CAST(NULL AS NULL)
```

Comment:
Comment is not clear. Please contrast the behavior before and after override clearly.

Reply:
updated the documentation. Please review it again when you get a chance.